WordPress/Classes/ViewRelated/NUX/ContentViews/Editor/UnifiedPrologueNotificationsContentView.swift

```diff
@@ -19,7 +19,7 @@ struct UnifiedPrologueNotificationsContentView: View {
                 RoundRectangleView {
                     HStack {
                         NotificationIcon(image: Appearance.topImage, size: notificationIconSize)
-                        Text(string: Appearance.topElementTitle)
```

Comment:
These initializers should use the `string:` name, as they're using a custom initializer added to `Text` (`init(string:boldMarker:)` – is it possible to remove these changes from this PR?

Reply:
Hey @frosty! Saw your [comment](https://github.com/wordpress-mobile/WordPress-iOS/pull/16201#discussion_r605140530) in a different PR by @leandroalonso that's been merged to develop. I'll merge develop into this PR 😄

Reply:
Thanks!